ArduinoCode/Pedal_Potentiometers/Pedal_Potentiometers.ino

```diff
@@ -88,10 +97,57 @@ void readSwitches() {
   brakePin1Val = digitalRead(brakePin1);
   brakePin2Val = digitalRead(brakePin2);
 
-  if ((brakePin1Val == LOW) && (brakePin2Val == LOW)) {
-    unsigned char valueMessage[8] = {1, 0, 0, 0, 0, 0, 0, 0};
-    CAN.sendMsgBuf(CAN_BRAKE, 0, 8, valueMessage); // send message that the switches are turned on
-  } else if (brakePin1Val != brakePin2Val) {
-    CAN.sendMsgBuf(CAN_BRAKE_BROKE, 0, 8, errorMessage); // send message that the two switch values are not equal
+  if ((brakePin1Val == LOW) && (brakePin2Val == LOW)) {  // switches are on
+    unsigned char valueMessage[1] = {1};
+    CAN.sendMsgBuf(CAN_BRAKE, 0, 1, valueMessage); // send message that the switches are turned on
+  } 
+  else if (brakePin1Val != brakePin2Val) {  // sends error message if brake pins don't agree
+    unsigned char valueMessage[4] = {'P', 0, 5, 04};  // error message of "Brake Switch A / B Correlation"
+    CAN.sendMsgBuf(CAN_BRAKE_BROKE, 0, 4, valueMessage);
+  }
+  else {    // switches  are off
+    unsigned char valueMessage[1] = {0};
+    CAN.sendMsgBuf(CAN_BRAKE, 0, 1, valueMessage); // send message that the switches are turned off
   }
 }
+
+/*
+ * readDashboard() receives two can ID's for motor on/off, and forward/reverse in order to be used in other functions.
+ * 0x01 holds the message whether the motor is on/off and stores the value in a global boolean variable - invertorOn.
+ * 0x02 holds the message whether in forward/reverse and stores the value in a global boolean variable - isForward.
+ */
+
+void readDashboard() {
+    if(CAN_MSGAVAIL == CAN.checkReceive()) {   //if a new message has been recieved. 
+      CAN.readMsgBuf(&len, buf); //enters message into program
+      canId = CAN.getCanId(); //gets sender ID
+     if (canID == 0x01) {  //determine whether Dashboard is on or off
+      MotorOff();
+      if (buf[0] == 0) {  // motor is off
+       invertorOn = false;
```

Comment:
```suggestion
    if(CAN_MSGAVAIL == CAN.checkReceive()) {   // if a new message has been recieved. 
      CAN.readMsgBuf(&len, buf); // enters message into program
      canId = CAN.getCanId(); // gets sender ID
     if (canID == 0x01) {  // determine whether Dashboard is on or off
      MotorOff();
      if (buf[0] == 0) {  // motor is off
       invertorOn = false;
```
Can you add a comment after "MotorOff();" to quickly say what it's doing? Something like "tells motor to turn off"